packages/core/src/fhir-deduplication/resources/encounter.ts

```diff
@@ -120,3 +125,52 @@ export function groupSameEncounters(encounters: Encounter[]): {
     danglingReferences,
   };
 }
+
+export function groupSameEncountersDateOnly(encounters: Encounter[]): {
```

Comment:
this is a copy paste of the above -> rip out the provider key stuff -> ensure we retain the most severe encounter class.

Reply:
Add TSDoc on both cross-referencing them so we can merge them into one when updating the original one, including your strategy for it